src/azul/chalice.py

```diff
@@ -195,8 +195,8 @@ def _api_gateway_context_middleware(self, event, get_response):
 
     hsts_max_age = 60 * 60 * 24 * 365 * 2
 
-    # Headers added to every response from an AWS Lambda function, as well as
-    # 4XX and 5XX responses from the API. Use of these headers prevent known
+    # Headers added to every response from the app, as well as canned 4XX and
+    # 5XX responses from API Gateway. Use of these headers addresses known
```

Comment:
The non-semantic changes should be moved to a `fixup` that immediately follows the 3rd commit (where this paragraph was first added).